back-end/src/main/kotlin/Gneiss/PacketCompiler/Service/Users.kt

```diff
@@ -11,26 +13,23 @@ class CreateUserRequest(
         var roleId: Int,
 )
 
-class CreateUserResponse(
-        var responseString: String
-)
-
 class Users(userDao: UserDao) {
     var userDao = userDao
 
-    fun createUser(req: CreateUserRequest): CreateUserResponse {
+    fun createUser(req: CreateUserRequest): ResponseEntity<String> {
         //Start by checking if there is already a user with the email specified in the CreateAccountRequest
         //If there is create and return a HTTP error response
         //Assume there is an account, and if not set the flag to false with checkAccountExists()
-        var accountExistsFlag: Boolean = true
-        accountExistsFlag = userDao.checkAccountExists(req.email)
+        var accountExistsFlag: Boolean = userDao.checkAccountExists(req.email)
 
         if(accountExistsFlag == true) {
-            return CreateUserResponse("Account already exists for this email")
+            return ResponseEntity<String>("Account already exists for this email", HttpStatus.NOT_ACCEPTABLE)
+            //return CreateUserResponse("Account already exists for this email")
         } else {
             //Create a new account and return a good response code
             userDao.createAccount(req.email, req.password, req.firstName, req.lastName, req.roleId)
-            return CreateUserResponse("User account created successfully")
+            return ResponseEntity<String>("User account created successfully", HttpStatus.OK)
+            //return CreateUserResponse("User account created successfully")
```

Comment:
Did you mean to leave these?

Reply:
I still need to fix some linter issues so I'll remove these in that commit